tensorflow/inference/docker/build_artifacts/sagemaker/python_service.py

```diff
@@ -412,4 +436,4 @@ def add_routes(self, application):
 
 app = falcon.API()
 resources = ServiceResources()
-resources.add_routes(app)
+resources.add_routes(app)
```

Comment:
As best practices. would put a new line at the bottom of this file. notice the red circle.

Reply:
Sure I will add a new line at the bottom of the file.